components/_global-navigation.scss

```diff
@@ -221,19 +252,23 @@ $notification-counter-size: 18px;
 
 	&__search-input {
 		border: 0;
+		box-sizing: border-box;
 		color: $color-dark-gray;
 		// Safari doesn't set default flex values if we use `flex: 1;`
 		// it requires us to put these values explicitly.
 		flex: 1 1 auto;
 		font-size: $typescale-size-minus-1;
-		// Safari doesn't trigger focus on input if it has `width: 0;`
-		margin-left: -1px;
-		// Firefox doesn't size input properly if our input
-		// has `width` set in `%` and is inside `flexbox`.
-		min-width: 0;
-		padding: 0;
-		// Safari doesn't trigger focus on input if it has `width 0;`
-		width: 1px;
+		// iPad Safari needs the height below to be set
+		height: 100%;
+		left: 0;
+		max-width: 100%;
+		padding: $search-input-padding;
+		padding-left: 20px;
+		position: absolute;
+		right: 0;
+		top: 0;
+		transition-duration: $search-transition-duration;
+		transition-property: width, padding;
```

Comment:
I've checked it again and it appeared that we don't need animating width any more. so we'll remove it.